public/index.html

```diff
@@ -39,5 +38,15 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
-  </body>
-</html>
+
+  <!-- The core Firebase JS SDK is always required and must be listed first -->
+  <script src="/__/firebase/7.4.0/firebase-app.js"></script>
```

Comment:
When I did this before I didn't need to include any js files, can you link to documentation for this?

Reply:
This isn't necessary for hosting but will be for almost all other fb functionality. I just included it as part of the Firebase project init. The snippet is found in Project Overview -> 1 App -> bt-web Settings. If the other fb features (ie analytics) aren't an immediate interest then we can remove it